helm_deploy/hmpps-it-incident-hub/values.yaml

```diff
@@ -70,6 +70,24 @@ generic-service:
     cloudplatform-live1-1: "35.178.209.113/32"
     cloudplatform-live1-2: "3.8.51.207/32"
     cloudplatform-live1-3: "35.177.252.54/32"
```

Comment:
DXC & MOJ IP addresses Match with Welcome people into prisons

Checked DOM1 IP addresses against Licences and the two added are listed as below:

ark-dom1-farnborough-psn: "51.247.3.0/24"
ark-dom1-corsham-psn: "51.247.4.0/24"

The 5 listed below are additional on licences, but I'm not sure if they are required for this product?

ark-dom1-ttp1: "195.59.75.0/24"
ark-dom1-farnborough: "194.33.192.0/24"
ark-dom1-corsham: "194.33.196.0/24"
ark-dom1-non-live-1: "194.33.193.0/25"
ark-dom1-non-live-2: "194.33.197.0/25"

Reply:
Just editing as some replicate what's already there with different names